### PR TITLE
[DP-172] Add home page for CWKW collection at "/cwkw"

### PR DESCRIPTION
**Pull request by chanmioh**
### Changelog
Moved all CWKW related files into `website/src/pages/cwkw` folder. Also added a TOC sidebar with dummy data for now. See images below of the new front-end changes.

image
- `index.page.tsx`: Starting home page of CWKW collection. 
    - Added a prototype collapsible table of contents sidebar to the left side of the screen (I don’t think this will be on the home page in the actual deployment, but I added this to show what I thought the TOC could look like.)
- `index.page.css.ts`: Added some styles in consideration of a collapsible TOC panel, such as shifting the main content to the right side of the screen.
- `theme.css.ts`: Just modified a few colors here so it’s more readable, but I assume this won’t be finalized yet.
- `toc-sidebar.tsx`: 
    - Created a desktop panel and mobile panel view.
    - Added styling with `toc-sidebar.css.ts`.

> Sidebar when closed:
image

> Sidebar when open:
image

> Mobile sidebar when closed: (For now, I made the button to open the drawer a button at the bottom right of the screen since I wasn't really sure where to put it... But this can be changed, I'm really open to suggestions)
image

> Mobile sidebar when open:
image

### CODE REVIEW

**Comment by loafofpiecrust** [comment]
Since we decided not to have a menu apart from the table of contents, for mobile the menu button at the top can open it.

**Comment by loafofpiecrust** [comment]
If this home page is gonna have the TOC as part of the page, i'm thinking it won't need the TOC as a drawer as well.

**Comment by chanmioh** [comment]
Changelog
- `index.page.tsx`: TOC drawer starts out expanded, and can be collapsed.

image

- `cwkw-layout.tsx`: 
    - Menu items are removed.
    - On mobile, the drawer icon can be seen and opened to reveal the TOC drawer.
 
image

image

- `toc-sidebar.tsx`: 
    - Removed mobile icon from the bottom right.

File: website/src/pages/cwkw/toc-sidebar.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  isOpen: boolean
+  setIsOpen: (bool: boolean) => void
+}) => {
+  return (
+    // If the sidebar is open, give its container more space in the flexbox.
+    <div style={{ flex: p.isOpen ? 2 : 1 }}>
+      <nav className={css.sidebar}>
+        {/* TOC Header  */}
+        <div className={css.tocHeader}>
+          {p.isOpen ? (
```
Comment (loafofpiecrust) [inline website/src/pages/cwkw/toc-sidebar.tsx:18]: Reminder to use the existing TOC component

**Comment by chanmioh** [comment]
### Changelog
- `toc.page.tsx` and `cwkw-layout.tsx`: Replaced usage of my own `<Chapters />` component with preexisting `<TOC />`.
- `toc-sidebar.css.tsx`: Made sure mobile nav button at top left shows when TOC desktop sidebar no longer does. 

**Current Appearance of TOC (Need to change colors later)**
image

**On mobile:**
image

**Review by loafofpiecrust — APPROVED** [review]

**Comment by chanmioh** [comment]
Picture of nested chapters content showing in TOC: 

image

**Pull request merged**